### PR TITLE
Fixing Rakefile

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

**Comment by AlecRaeside** [comment]
LGTM

File: Rakefile [hunk 1]
```diff
@@ -26,8 +24,6 @@ namespace :test do
       `rm #{css}.map` if File.file?("#{css}.map")
     end
 
-    `rm test.scss` if File.file?('test.scss')
```
Comment (kllevin) [inline Rakefile:29]: I assume you aren't removing style-test.scss?

**Comment by kllevin** [comment]
LGTM

**Comment by smgcode** [comment]
LGTM

File: Rakefile (hunk 1 above)
Comment (chris-pearce) [inline Rakefile:29]: No that guys stays as it's needed for me to work on WFC.

**Pull request merged**